_config.yml

```diff
@@ -2,7 +2,7 @@ baseurl: ''
 exclude: [vendor, deploy_key.enc, TEMPLATE.shtml, archive_html, dev_website]
 
 # Google analytics
-google_analytics: G-5FBPVMZYDF
+google_analytics:
```

Comment:
Can you explain what happens with the included analytics jscript if the ID is missing? Does it disable it, or do you know if it generates any errors?

Reply:
@jasoncpatton In the head there is a check for the presence of the tag, and if it is not present then it skips over the include of the analytic related code.

Reply:
https://github.com/CHTC/chtc-website-source/blob/master/_layouts/default.html#:~:text=%7B%25%20if%20site.google_analytics,%7B%25%20endif%20%25%7D

Reply:
https://github.com/CHTC/chtc-website-source/blob/596c041e0b0ad21e9c785156ff38f961ca1154a7/_layouts/default.html#L21-L23